Sources/GoodSwiftUI/ReadableContentWidthModifier.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ReadableContentWidthModifier.swift
+//
+//  GoodSwiftUI
+//  Created by Filip Šašala on 31/12/2023.
+//
+
+import SwiftUI
+
+private struct ReadableContentWidth: ViewModifier {
+
+    private let measureViewController = UIViewController()
```

Comment:
This is created each time parent view calls this modifier. Not a big deal, but we should keep that in mind. I would use static property and keep one single instance of dummy controller for the whole app.